chainercv/links/model/faster_rcnn/faster_rcnn_vgg.py

```diff
@@ -74,7 +73,8 @@ class FasterRCNNVGG16(FasterRCNN):
         'voc07': {
             'n_fg_class': 20,
             'url': 'https://github.com/yuyu2172/share-weights/releases/'
-            'download/0.0.3/faster_rcnn_vgg16_voc07_2017_06_06.npz'
+            'download/0.0.4/'
+            'faster_rcnn_vgg16_voc07_trained_2017_08_06_trial_4.npz'
```

Comment:
Perhaps, `faster_rcnn_vgg16_voc07_2017_08_06.npz` is better. User will think "what is trial_4?" like me.